From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Haris Okanovic <haris.okanovic@ni.com>
Date: Mon, 21 Jul 2014 13:49:18 -0500
Subject: [PATCH] Updated verifyconnect() for PharLAP

Windows builds of verifyconnect() yield by calling SleepEx(0), but PharLAP
does not have that system function, so we're using Sleep(0) instead.

Upstream-Status: Inappropriate [Pharlap specific]

diff --git a/lib/connect.c b/lib/connect.c
index b000b1b2c..87cfc2679 100644
--- a/lib/connect.c
+++ b/lib/connect.c
@@ -518,6 +518,8 @@ static bool verifyconnect(curl_socket_t sockfd, int *error)
 
 #ifdef _WIN32_WCE
   Sleep(0);
+#elif defined(PHARLAP)
+  Sleep(0);
 #else
   SleepEx(0, FALSE);
 #endif
-- 
2.28.0

